docs/guides/connecting_to_your_data/cloud/s3/spark.md

```diff
@@ -32,15 +33,7 @@ Import these necessary packages and modules.
 ```python file=../../../../../tests/integration/docusaurus/connecting_to_your_data/cloud/spark_s3_yaml_example.py#L1-L4
 ```
 
-:::note Load your DataContext into memory
-
-Use one of the guides below based on your deployment:
-
-- [How to instantiate a Data Context without a yml file](/docs/guides/setup/configuring-data-contexts/how-to-instantiate-a-data-context-without-a-yml-file)
-- [How to instantiate a Data Context on an EMR Spark cluster](/docs/deployment_patterns/how-to-instantiate-a-data-context-on-an-emr-spark-cluster)
-- [How to instantiate a Data Context on Databricks Spark cluster](/docs/deployment_patterns/how-to-instantiate-a-data-context-on-databricks-spark-cluster)
-
-:::
+<SparkDataContextNote />
```

Comment:
Thanks for splitting this out.  I dont think this is a point we need to address now, but once these docs are written I think they can be merged into the 3 tabs that are part of step 1)

- No CLI + filesystem tab --> DataContext without yml file
- No CLI + no filesystem --> DataContext in EMR / Spark cluster

Reply:
Jira task has been created for this. Thanks for the discussion @Aylr